073-decouple-ca-management-from-end-entity-certificate-management.md

```diff
@@ -0,0 +1,339 @@
+The Cluster Operator accomplishes this by using `openssl` to generate self-signed root CA certificates and private keys which it then uses to directly sign end-entity (EE) certificates (i.e. intermediate certificates are not used).
+A _cluster CA_  and _clients CA_ are generated.
+These CAs are only used for Kafka clusters and a unique instance of each CA is used for each Kafka cluster.
+The cluster CA issues certificates for the following Strimzi components:
+* ZooKeeper nodes
+* Kafka nodes
+* User and Topic operators
+* Cruise Control
+* Kafka Exporter
+
+The clients CA issues certificates for user applications using the User Operator, or through another external mechanism chosen by the user.
+
+The cluster CA root certificate is added to the trust stores of Strimzi components.  This ensures that, for example, ZooKeeper nodes trust each other's certificates as well as the certificates of brokers, and brokers trust certificates issued to other brokers.
+
+#### Fig 1: Cluster CA certificate and component relationships
+
+![Cluster CA relationships](./images/073-cluster-ca.svg)
+
+> Fig 1: In the diagram, the red lines show trust.
+> For example, because the Kafka broker trusts the Cluster CA certificate,
+> and the Cluster CA certificate signed the ZK EE certificate
+> the broker will trust the ZK EE certificate presented by the ZK node
+> during TLS handshake.
+
+The cluster CA root certificate also needs to be trusted by Kafka clients connecting to the cluster so that clients trust the certificates presented by the brokers.
+The clients CA root certificate is added to the broker trust stores too, so that the brokers will trust certificates issued to Kafka client applications.
+
+#### Fig 2: Clients CA certificate and component relationships
+
+![Clients CA relationships](./images/073-clients-ca.svg)
+
+### Existing Kubernetes Secrets
+
+Strimzi currently uses the following Kubernetes Secrets to store the CA certificates and private keys:
+* <CLUSTER_NAME>-cluster-ca-cert for the Cluster CA root certificate
+* <CLUSTER_NAME>-cluster-ca for the Cluster CA private key
+* <CLUSTER_NAME>-clients-ca-cert for the Clients CA root certificate
+* <CLUSTER_NAME>-clients-ca for the Clients CA private key
+
+Strimzi currently uses the following Kubernetes Secrets to store server certificates (signed by the cluster CA) that components will present to clients:
+* <CLUSTER_NAME>-kafka-brokers for Kafka brokers
+* <CLUSTER_NAME>-zookeeper-nodes for ZooKeeper nodes
+
+Strimzi currently uses the following Kubernetes Secrets to store client certificates (signed by the cluster CA) that components will present to Kafka or ZooKeeper when making requests:
+* <CLUSTER_NAME>-cluster-operator-certs for the Cluster Operator
+* <CLUSTER_NAME>-entity-topic-operator-certs for the Entity Topic Operator
+* <CLUSTER_NAME>-entity-user-operator-certs for the Entity User Operator
```

Comment:
```suggestion
* <CLUSTER_NAME>-entity-user-operator-certs for the User Operator
```

Reply:
I know that TO and UO are part of the EO but we usually refer to them just as TO and UO and not ETO and EUO.